frontend/src/Frontend/Page/Transaction.hs

```diff
@@ -55,21 +58,40 @@ transactionPage
 transactionPage netId cid bp = do
     let txs = _blockPayloadWithOutputs_transactionsWithOutputs bp
     el "h2" $ text $ (tshow $ length txs) <> " Transactions"
-    divClass "ui accordion" $ do
+    divClass "ui" $ do
       forM_ txs $ \(t, tout) -> mdo
-        open <- toggle False $ domEvent Click e
+        -- open <- toggle False $ domEvent Click e
```

Comment:
The reason I had them collapse was so you could quickly see how many transactions are in the block when there is more than one.  Any thought on how to still achieve this?